### PR TITLE
REPO-3982 Add NGINX

### PR DESCRIPTION
**Pull request by killerboot**

### CODE REVIEW

File: docker-alfresco/test/docker-alfresco-all-amps-test/test/docker-compose.yml [hunk 1]
```diff
@@ -143,6 +147,24 @@ services:
             - 61616:61616 # OpenWire
             - 61613:61613 # STOMP
 
+    digital-workspace:
+        image: quay.io/alfresco/alfresco-digital-workspace:${DIGITAL_WORSPACE_TAG}
+
+    digital-workspace-ingress:
```
Comment (derekhulley) [inline docker-alfresco/test/docker-alfresco-all-amps-test/test/docker-compose.yml]: This name will be changed.

**Comment by derekhulley** [comment]
https://issues.alfresco.com/jira/browse/REPO-3982 completed

**Comment by derekhulley** [comment]
@killerboot please delete this branch if there is nothing more we need.

**Comment by killerboot** [comment]
@derekhulley This can still serve an example for https://issues.alfresco.com/jira/browse/REPO-4000, see last acceptance criteria.

**Pull request closed without merging**